readthedocs/doc_builder/exceptions.py

```diff
@@ -52,6 +52,13 @@ class BuildUserError(BuildBaseException):
         "and it is not currently supported. "
         'Please, remove all the files but the "{artifact_type}" your want to upload.'
     )
+    PDF_COMMAND_FAILED = gettext_noop(
+        "PDF generation failed. "
+        "Please double check the command's output looking for errors and fix them. "
+        "Note it's possible the PDF has been always failing silently in your project, "
+        "but due to latest changes, "
+        "Read the Docs now makes it explicit and fail the whole build."
+    )
```

Comment:
```suggestion
    PDF_COMMAND_FAILED = gettext_noop(
        "PDF generation failed. "
        "The build log below contains information on what errors caused the failure." 
        "Our code has recently changed to fail the entire build on PDF errors, "
        "where we used to pass the build when a PDF was created."
        "Please contact us if you need help understanding this error."
    )
```

I don't love this. I think we probably want a link to a docs article here, but this is much better than a generic build error 👍

Reply:
Yeah, but it works for now 😄 . Let's move forward with this and update the message as we go.